packages/client/src/client.spec.js

```diff
@@ -1,83 +1,68 @@
 'use strict';
 const baseUrl = 'http://localhost:4010/';
 
+const testRequest = (request, statusCode) => {
+  const sgClient = require('./client');
+  sgClient.setApiKey('SendGrid API Key');
+  sgClient.setDefaultRequest('baseUrl', baseUrl);
+  sgClient.setDefaultHeader('X-Mock', statusCode);
+  return sgClient
+    .request(request)
+    .then(([response, body]) => {
+      expect(response.statusCode).to.equal(statusCode);
+    });
+};
+
 /**
  * Tests
  */
 describe('test_access_settings_activity_get', () => {
   const request = {};
-  const queryParams = {
-    'limit': 1,
+  request.qs = {
+    limit: 1,
   };
-  request.qs = queryParams;
   request.method = 'GET';
   request.url = '/v3/access_settings/activity';
   it('should have the correct response code', () => {
-    const sgClient = require('./client');
-    sgClient.setApiKey('SendGrid API Key');
-    sgClient.setDefaultRequest('baseUrl', baseUrl);
-    sgClient.setDefaultHeader('X-Mock', 200);
-    return sgClient
-      .request(request)
-      .then(([response, body]) => {
-        expect(response.statusCode).to.equal(200);
-      });
+    return testRequest(request, 200);
   });
 });
 
 describe('test_access_settings_whitelist_post', () => {
   const request = {};
-  const data = {
-    'ips': [
+  request.body = {
+    ips: [
```

Comment:
@childish-sambino did you remove quotes manually or did you use a tool/linter?

Reply:
Told IntelliJ to fix all occurrences in the file.